docs/usage/system_backend/wrapping.rst

```diff
@@ -42,6 +42,17 @@ Examples
     )
     print('Unwrapped approle role token secret id accessor: "%s"' % unwrap_response['data']['secret_id_accessor'])
 
+.. testcode:: sys_wrapping
+
+    import hvac
+
+    client = hvac.Client(url='https://127.0.0.1:8200')
+    client.token = "s.xxxxxxxxxxxxxxxxxxxxxx"
+
+    # When authenticating with just the wrapping token, should not pass token into unwrap call
```

Comment:
suggestion: I think this could be reworded better

```suggestion
    # Do not pass the token to unwrap when authenticating with the wrapping token
```

Reply:
Updated.